biocypher/write/graph/_neo4j.py

```diff
@@ -285,7 +285,11 @@ def _construct_import_call(self) -> str:
         import_call_neo4j_v5 = self._get_import_call(
             "database import full", "", "--overwrite-destination="
         )
-        import_script = f"# Neo4j v4 import call:\n{import_call_neo4j_v4}\n# Neo4j v5 import call:\n{import_call_neo4j_v5}\n"
+        neo4j_verison_check = (
+            f"version=$(neo4j-admin --version | cut -d '.' -f 1)"
+        )
+        output_command = '"${cmd[@]}"'
+        import_script = f"#!/bin/bash\n# Check Neo4j version\n{neo4j_verison_check}\nif [ $version -ge 5 ]; then\n\t # Neo4j v5 import call:\n\t{import_call_neo4j_v5}\nelse\n\t# Neo4j v4 import call:\n{import_call_neo4j_v4}\nfi\n{output_command}"
```

Comment:
Have you tested the whole import script?
I just tested it with one of the small tutorial examples within BioCypher. After copying the script to the DB root and running it I get `./import.sh: line 4: [: too many arguments`.
The import script looks like this:
```
#!/bin/bash
# Check Neo4j version
version=$(neo4j-admin --version | cut -d '.' -f 1)
if [ $version -ge 5 ]; then
	 # Neo4j v5 import call:
	bin/neo4j-admin database import full --delimiter=";" --array-delimiter="|" --quote="'" --overwrite-destination=true --nodes="pathes" relationships 
else
	# Neo4j v4 import call:
bin/neo4j-admin import --delimiter=";" --array-delimiter="|" --quote="'" --force=true --nodes="pathes" --nodes=" --database=relationships 
fi
"${cmd[@]}"
```